src/SDK/Language/Swift.php

```diff
@@ -284,6 +284,11 @@ public function getFiles(): array
                 'destination'   => '/Sources/{{ spec.title | caseUcfirst}}Models/{{ definition.name | caseUcfirst }}.swift',
                 'template'      => '/swift/Sources/Models/Model.swift.twig',
             ],
+            [
+                'scope' => 'enum',
+                'destination' => '/Sources/{{ spec.title | caseUcfirst}}/Enums/{{ enum.name | caseUcfirst }}.swift',
```

Comment:
We should make this into another module, so the path would be:
```php
                'destination' => '/Sources/{{ spec.title | caseUcfirst}}Enums/{{ enum.name | caseUcfirst }}.swift',
```
Additionally, the Package.swift would need to be updated to declare the new target and add it as a dependency to the main Appwrite module. The service class would need to `import AppwriteEnums`